### ISSUE TITLE
'import' and 'export' may only appear at the top level

### ISSUE THREAD

**Issue by goooseman** [rwatts3/orionjs-cloudinary#5]
Just try it on the clean application.
Meteor v1.2.1

**Comment by romans127** [comment]
Hi Could you provide a few more details ? 

Full description, in relation to the title ? Also which version are you using. Notice the new version introduced breaking changes as it is `v1.0.0` The configuration is now done by the meteor settings file. In preparation for `meteor 1.3`. I foresee major changes coming to Orion, and the way the previous version handled the settings could cause more trouble down the line.

**Comment by goooseman** [comment]
To reproduce the problem:

```
meteor create test
cd test
meteor add rwatts:orionjs-cloudinary
meteor
```

Then you will get:

```
$ meteor
[[[[[ ~/dev/test ]]]]]

=> Started proxy.
=> Started MongoDB.
=> Errors prevented startup:

   While computing assigned variables:
   /packages/rwatts_orionjs-cloudinary/cloudinary.js: 'import' and 'export' may only appear at the top level (51:0)

   While computing assigned variables:
   /packages/rwatts_orionjs-cloudinary/cloudinary.js: 'import' and 'export' may only appear at the top level (51:0)
```

Yeah, you can create a `settings.json` file, but the problem is that this package can work only with Meteor v1.3, as far as I can see.

**Comment by romans127** [comment]
Ah I see,

Ok this package is only for `orionjs`. use `lepozepo:cloudinary` if you want
to use it in non Orion specific projects.

![](https://link.nylas.com/open/7hecocdyi1g0zt42ceol45sjc/8af5a0be15ff47a59957
bbce3b557993)

**Comment by goooseman** [comment]
No, on my project I'm using it with orion. But the problem is the same.

**Comment by romans127** [comment]
Intersting. 

Yes I suppose the projects I'm using it in are  1.3 , Out of curiosity do you know if there is a way to make modules and ecmascript a weak dependency ? If so I'd be happy to go that route for older versions of meteor.

**Comment by goooseman** [comment]
the problem is only in the last line
instead of `exports default Cloudinary`, u can just define Cloudinary object at the start of the file without `var` or `let`. It will be available everywhere in the project.

**Comment by romans127** [comment]
Gotcha, 

Ok if you have a PR and want to submit for credit, I'd be happy to do so, otherwise I can just test and push version `1.0.1`

**Comment by goooseman** [comment]
10 minutes

**Comment by romans127** [comment]
Sweet, 

Thank you.

**Comment by romans127** [comment]
Ok #6 Merged, and `v1.0.1` released to atmosphere.

Thank you for contributing.
